share/pot/common-flv.sh

```diff
@@ -1,5 +1,71 @@
 #!/bin/sh
 
+# $1 flavour name
+_is_flavour()
+{
+    # shellcheck disable=SC3043
+    local _flv_name
+	_flv_name="$1"
+	if [ -n "$( _get_flavour_script "$_flv_name" )" ] ||
+		[ -n "$( _get_flavour_cmd_file "$_flv_name" )" ]; then
+		return 0 # true
+	fi
+	return 1 # false
+}
+
+_get_flavour_script()
+{
+    # shellcheck disable=SC3043
+    local _flv_name
+	_flv_name="$1"
+	if [ -f "$_flv_name" ] && [ -x "$_flv_name" ] && [ "$_flv_name" != "${_flv_name%%.sh}" ]; then ## it's a script path name
```

Comment:
Nice idiomatic (and confusing) check: `[ "$_flv_name" != "${_flv_name%%.sh}" ]` - derailed me for a seconf